addon/controllers/building/edit/entrance/edit/link-street.js

```diff
@@ -21,16 +21,19 @@ export default class BuildingEditEntranceLinkStreetController extends Controller
   @tracked PLZ4 = null;
   @tracked PLZ6 = null;
 
-  get baseQuery() {
+  get baseModel() {
```

Comment:
Why did you rename it? I think the previous name was more accurate since its not really a model.

Reply:
done